src/hooks/useEngineConnectionSubscriptions.ts

```diff
@@ -186,6 +189,20 @@ export function useEngineConnectionSubscriptions() {
                 faceId,
                 kclManager.artifactGraph
               )
+              if (!err(extrusion)) {
+                const fileIndex = sourceRangeFileIndex(extrusion.codeRef.range)
+                if (fileIndex !== 0) {
+                  const importDetails =
+                    kclManager.execState.filenames[fileIndex]
+                  if (importDetails?.type === 'Local') {
```

Comment:
I wanted to check with the kcl oracle if I should checking anything else here, there's `main` and `std` but main is the happy path, and we shouldn't std is not ever geometry right?

there might one day be `url` or similar, but can probably update at the time?

Reply:
That's correct.

If you'd like the addition of a new case to fail `tsc`, you can make an exhaustive check like this:

https://github.com/KittyCAD/modeling-app/blob/d270447777582f97cbb27424ff7c30a4af5c556d/src/lib/operations.ts#L1126

Reply:
Done.